dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/calculate-fixed-fee.ts

```diff
@@ -53,18 +17,35 @@ const getNumberOfDaysRemainingInCoverPeriod = (reportPeriodStart: MonthAndYear,
   return differenceInDays(coverEndDate, currentReportPeriodStartMonthStart);
 };
 
+export type CalculateFixedFeeParams = {
+  utilisation: number;
+  reportPeriod: ReportPeriod;
+  coverStartDate: Date;
+  coverEndDate: Date;
+  interestPercentage: number;
+  dayCountBasis: number;
+};
+
 /**
- * Calculates the fixed fee for the given report period
- * @param utilisation - The facility utilisation
- * @param facilityId - The ukef facility id
- * @param reportPeriod - The report period
+ * Calculates the fixed fee for the given parameters
+ * @param param - The parameters to calculate the fixed fee with
+ * @param param.utilisation - The facility utilisation
+ * @param param.reportPeriod - The report period
+ * @param param.coverStartDate - The facility cover start date
+ * @param param.coverEndDate - The facility cover end date
+ * @param param.interestPercentage - The facility interest percentage
+ * @param param.dayCountBasis - The facility day count basis
  * @returns The fixed fee for the current report period
  */
-export const calculateFixedFee = async (utilisation: number, facilityId: string, reportPeriod: ReportPeriod): Promise<number> => {
-  const { coverEndDate, coverStartDate, dayCountBasis, interestPercentage } = await getLatestTfmFacilityValues(facilityId);
-
+export const calculateFixedFee = ({
+  utilisation,
+  reportPeriod,
+  coverStartDate,
+  coverEndDate,
+  interestPercentage,
```

Comment:
Interest percentage here means the decimal representation but in the other files and on the facility it means out of a 100, I personally think percentage should be out of 100 one so I would probably move the dividing by 100 into here or name this slightly differently (and/or could be explicit in the docstring perhaps?)

Reply:
I've just performed the "divide by 100" in this function as keeping the format of `interestPercentage` consistent will probably be easier to understand